packages/components/src/circular-option-picker/stories/index.story.tsx

```diff
@@ -5,7 +5,7 @@ import type { Meta, StoryFn } from '@storybook/react';
 /**
  * WordPress dependencies
  */
-import { useState, createContext, useContext } from '@wordpress/element';
+import { createContext, useContext, useState } from '@wordpress/element';
```

Comment:
Suggestion (non-blocking): This could be due to an ESLint or Prettier configuration, but I recommend avoiding unrelated changes in the future.